cpmpy/solvers/pysat.py

```diff
@@ -130,7 +133,7 @@ def solvernames():
                 names.append(name)
         return names
 
-    def __init__(self, cpm_model=None, subsolver=None):
+    def __init__(self, cpm_model=None, subsolver=None, encoding="auto"):
```

Comment:
Question @tias : I've just added and documented this encoding arg, but I'm not sure how we add "solver" (or rather transformation) specific configuration options. Perhaps it's best to leave out this argument and just always set the encoding to "auto" for now?

Advanced users can still change `PySAT.encoding = "direct"`, but then this does not become an expected part of the interface.

Reply:
I've removed this arg for now, see commit 4b56f926